internal/local/command_kafka_broker_describe.go

```diff
@@ -0,0 +1,70 @@
+package local
+
+import (
+	"context"
+	"strconv"
+
+	"github.com/spf13/cobra"
+
+	"github.com/confluentinc/cli/v3/pkg/broker"
+	pcmd "github.com/confluentinc/cli/v3/pkg/cmd"
+	"github.com/confluentinc/cli/v3/pkg/errors"
+	"github.com/confluentinc/cli/v3/pkg/examples"
+	"github.com/confluentinc/cli/v3/pkg/output"
+	"github.com/confluentinc/cli/v3/pkg/utils"
+)
+
+func (c *Command) newKafkaBrokerDescribeCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "describe [id]",
+		Short: "Describe a local Kafka broker.",
+		Long:  "Describe cluster-wide or per-broker configuration values.",
+		Args:  cobra.MaximumNArgs(1),
+		RunE:  c.brokerDescribe,
+		Example: examples.BuildExampleString(
+			examples.Example{
+				Text: `Describe the "min.insync.replicas" configuration for broker 1.`,
+				Code: "confluent local broker describe 1 --config-name min.insync.replicas",
+			},
+		),
+	}
+
+	cmd.Flags().String("config-name", "", "Get a specific configuration value")
```

Comment:
The linter should check for this, right?
```suggestion
	cmd.Flags().String("config-name", "", "Get a specific configuration value.")
```